package.json

```diff
@@ -12,11 +12,11 @@
     "generate:middleware-types": "graphql-codegen -r dotenv/config",
     "fetch-definitions": "cross-env node scripts/fetchDefinitions.js",
     "update-readme": "cross-env node scripts/updateReadme.js",
-    "generate:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --package polymesh-types --input ./src/polkadot",
-    "generate:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --package polymesh-types --endpoint ws://localhost:9944 --output ./src/polkadot --strict",
+    "generate:defs": "ts-node --skip-project node_modules/.bin/polkadot-types-from-defs --endpoint metadata.json --package polymesh-types --input ./src/polkadot/",
+    "generate:meta": "ts-node --skip-project node_modules/.bin/polkadot-types-from-chain --endpoint metadata.json --package polymesh-types --output ./src/polkadot --strict",
     "generate:tags": "cross-env node scripts/generateTxTags.js",
     "generate:post-process": "cross-env node scripts/postProcessTypes.js",
-    "test": "jest --detectOpenHandles --coverage",
+    "test": "jest --detectOpenHandles",
```

Comment:
Will add back in, but maybe we should have a `no-cov` variation. I find the coverage report annoying when debugging a test.